index.html

```diff
@@ -7,12 +7,16 @@
     <title>Document</title>
 </head>
 <body>
-    <main>
-        <header id="toolbar">
-            <h1>my logo</h1>
-            <img src="./Icons/Icon - Menu.svg" alt="">
-        </header>
+    <header id="toolbar">
+        <nav>
+            <ul>
+                <li><a href="/">my logo</a></li>
+                <li><button type="button"><img src="./Icons/Icon - Menu.svg" alt=""></button></li>
```

Comment:
- [Optional] Dear @abdullah97-pro, Usually the logo is not part of the navigation items list, hence should not be inside the navigation lists (`li`). It would be better to make your logo part of the header alongside the nav items.